src/scene/light.js

```diff
@@ -65,6 +65,9 @@ class LightRenderData {
         // scissor rectangle for the shadow rendering to the texture (x, y, width, height)
         this.shadowScissor = new Vec4(0, 0, 1, 1);
 
+        // depth range for cascaded shadows
```

Comment:
note: this is not only for cascaded shadows, but for all directional lights

Reply:
Fixed this comment.